### PR TITLE
test/NIAIDexternal_url: add external_url to etl mapping and potal gitops

### PR DESCRIPTION
**Pull request by ocshawn**
Link to Jira ticket if there is one:

### Environments
NIAID

### Description of changes
- add external_url to etl mapping
- add external_url to potal gitops download btn

### CODE REVIEW

**Review by paulineribeyre — APPROVED** [review]

File: qa-niaid.planx-pla.net/portal/gitops.json [hunk 1]
```diff
@@ -130,7 +130,8 @@
       "defaultOrderBy": ["title", "asc"],
       "buttons": [
         {
-          "type": "download"
+          "type": "download",
+          "overrideUrlField": "external_url"
```
Comment (paulineribeyre) [inline qa-niaid.planx-pla.net/portal/gitops.json:134]: aw, we should have called `required_idp_field` => `requiredIdpField`

Comment (paulineribeyre) [reply qa-niaid.planx-pla.net/portal/gitops.json:134]: do you think we could fix this inconsistency?

Comment (ocshawn) [reply qa-niaid.planx-pla.net/portal/gitops.json:134]: ill make the change as well, just will need to remember to update the gitops after i push the portal changes

Comment (paulineribeyre) [reply qa-niaid.planx-pla.net/portal/gitops.json:134]: thanks

**Pull request merged**